packages/devtools/client/components/ServerRouteDetails.vue

```diff
@@ -308,6 +308,50 @@ watchEffect(() => {
   }
 })
 
+const savedRouteInputs = useLocalStorage<{ path: string, tab: string, inputs: any }[]>('nuxt-devtools:server-routes:inputs', () => [], {
+  window: window.parent,
```

Comment:
Is there a reason to pass the parent window here? It should work with iframe window as they are same origin.

Reply:
While trying out with my laptop(windows 10), the storage wasn't working as expected(the client would get reset after a page refresh)
Is it possible for you to test it out?